vmm/src/vmm_config/instance_info.rs

```diff
@@ -38,6 +38,8 @@ pub struct InstanceInfo {
     pub id: String,
     /// The state of the microVM.
     pub state: InstanceState,
+    /// The version of the Firecracker container that runs the microVM
+    pub firecracker_version: String,
```

Comment:
Can you rename `firecracker_version` to something not Firecracker-specific?

Reply:
I renamed it to vmm_version. I don't know if it's the best name.